apps/opik-backend/src/test/java/com/comet/opik/api/resources/v1/priv/ExperimentsResourceTest.java

```diff
@@ -1872,6 +1937,20 @@ void createAndGetWithDeletedTrace() {
         }
     }
 
+    private PromptVersion createPromptVersion(CreatePromptVersion promptVersion, String apiKey,
+            String workspaceName) {
+        try (var response = client.target(PROMPT_PATH.formatted(baseURI) + "/versions")
+                .request()
+                .header(HttpHeaders.AUTHORIZATION, apiKey)
+                .header(RequestContext.WORKSPACE_HEADER, workspaceName)
+                .post(Entity.json(promptVersion))) {
+
+            assertThat(response.getStatus()).isEqualTo(org.apache.hc.core5.http.HttpStatus.SC_OK);
+
+            return response.readEntity(PromptVersion.class);
+        }
+    }
+
```

Comment:
Can we move it to some Utility class? We have the same code in `DatasetsResourceTest` and in `PromptResourceTest`

Reply:
Added, however, I didn't refactor the other test to avoid increasing too much the scope of this PR